pkg/crc/machine/bundle/constants.go

```diff
@@ -8,23 +8,23 @@ import (
 var bundleLocations = map[string]bundlesDownloadInfo{
 	"amd64": {
 		"darwin": {
-			preset.OpenShift: download.NewRemoteFile("https://storage.googleapis.com/crc-bundle-github-ci/4.9.12/crc_hyperkit_4.9.12.crcbundle",
-				"73167e735337b24efa18654667d8bff7cb2b7eb2d0bd2c6ffae4f1df4d8f6ced"),
+			preset.OpenShift: download.NewRemoteFile("https://storage.googleapis.com/crc-bundle-github-ci/4.9.15/crc_hyperkit_4.9.15.crcbundle",
+				"9e90bcbce5837cfc7c13f94d269fe043ba38c3228b57ba4748cc52d47ededf86"),
```

Comment:
why is this part of the change?

Reply:
@gbraad it is part of different commit message to make CI happy.

Reply:
ugh...
 OK